src/store/permissions/reducer.js

```diff
@@ -0,0 +1,39 @@
+import { applyReducerHash } from '@redhat-cloud-services/frontend-components-utilities/files/ReducerRegistry';
+import { ACTION_TYPES } from '../../constants';
+
+const loadWritePermissionsPending = (state) => ({
+    ...state,
+    loadingFailed: false,
+    loading: !insights.chrome.isProd,
+    writePermissions: insights.chrome.isProd
+});
+
+const loadWritePermissionsFulfilled = (state, { payload }) => ({
+    ...state,
+    loading: false,
+    loadingFailed: false,
+    writePermissions: payload.writePermissions
+});
+
+const loadWritePermissionsFailed = (state) => ({
+    ...state,
+    loading: false,
+    loadingFailed: true
+});
+
+const defaultPermissionState = {
+    loading: !insights.chrome.isProd,
+    writePermissions: insights.chrome.isProd,
```

Comment:
I would change these default values to
```suggestion
    loading: false,
    writePermissions: undefined,
```

Reply:
Actually no, because when you are on prod, there is no loading happening (because we don't have to do it - it would be just wasting of users's resources - on prod they have always write permissions now)